javascript/modules/client/src/internal/interfaces/dao.ts

```diff
@@ -128,3 +128,8 @@ export enum VoteOption {
   YEA,
   NAY,
 }
+
+export interface IGasFeeEstimation {
+  avg: BigNumber;
```

Comment:
Being a general purpose SDK, I would tend to use clearer words
```suggestion
  average: BigNumber;
```